### PR TITLE
updated the code of factorial and coffee shop

### PR DESCRIPTION
**Pull request by harsh3294**
updated the code of factorial ==> if the user adds the negative value then it wont allow and it will display the invalid input 
updated the code of coffee shop ==> added the menu , also it ask the user to add the quantity

### CODE REVIEW

File: 02/HelloWorld/HelloWorld.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31515.178
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HelloWorld", "HelloWorld\HelloWorld.csproj", "{D618BE94-B9A5-4664-BB97-3D861F109FF4}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
```
Comment (kaurrevature) [inline 02/HelloWorld/HelloWorld.sln:12]: @harsh3294 unstage these files and make another PR

File: 02/HelloWorld/HelloWorld/HelloWorld.csproj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (kaurrevature) [inline 02/HelloWorld/HelloWorld/HelloWorld.csproj:5]: @harsh3294  remove this file

**Review by kaurrevature — CHANGES_REQUESTED** [review]

File: 02/HelloWorld/HelloWorld/HelloWorld.csproj (hunk 2 above)
Comment (harsh3294) [reply 02/HelloWorld/HelloWorld/HelloWorld.csproj]: removed file

**Review by kaurrevature — CHANGES_REQUESTED** [review]
There are 12 files added, only 3 files need to be committed. Please do the needful.

**Pull request closed without merging**